src/core/01_apim_0.tf

```diff
@@ -0,0 +1,114 @@
+# 🔐 KV
+data "azurerm_key_vault_secret" "apim_publisher_email" {
+  name         = "apim-publisher-email"
+  key_vault_id = data.azurerm_key_vault.kv.id
+}
+
+## 🎫  Certificates
+
+data "azurerm_key_vault_certificate" "apim_internal_certificate" {
+  name         = var.apim_api_internal_certificate_name
+  key_vault_id = data.azurerm_key_vault.kv.id
+}
+
+# 🧵 DNS ZONE
+data "azurerm_private_dns_zone" "internal" {
+  name                = "internal.${var.prod_dns_zone_prefix}.${var.external_domain}"
+  resource_group_name = data.azurerm_resource_group.rg_vnet.name
+}
+
+#--------------------------------------------------------------------------------------------------
+
+resource "azurerm_resource_group" "rg_api" {
+  name     = "${local.project}-api-rg"
+  location = var.location
+
+  tags = var.tags
+}
+
+# APIM subnet
+module "apim_snet" {
+  source               = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v2.1.21"
+  name                 = "${local.project}-apim-snet"
+  resource_group_name  = data.azurerm_resource_group.rg_vnet.name
+  virtual_network_name = data.azurerm_virtual_network.vnet.name
+  address_prefixes     = var.cidr_subnet_apim
+
+  enforce_private_link_endpoint_network_policies = true
+  service_endpoints                              = ["Microsoft.Web"]
+}
+
+###########################
+## Api Management (apim) ##
+###########################
+
+module "apim" {
```

Comment:
why do we need a template for apim?
we already have the module? it looks like the template of the template... 
Why not to put an example folder within the module itself like here:
https://github.com/terraform-aws-modules/terraform-aws-vpc/tree/master/examples

Reply:
we can remove the this file and the others to avoid to much noise